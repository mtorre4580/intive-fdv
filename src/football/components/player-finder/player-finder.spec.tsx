import React from 'react';
import PlayerFinder from './player-finder';
import renderer from 'react-test-renderer';

describe('<PlayerFinder />', () => {
   it('should render snapshot', () => {
    const props : any = {
        players: [],
        loading: false,
        error: false,
        onSearch: jest.fn()
    };
    const component = renderer.create(<PlayerFinder {...props} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
    
});