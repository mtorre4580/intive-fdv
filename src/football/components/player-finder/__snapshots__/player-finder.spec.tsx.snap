// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlayerFinder /> should render snapshot 1`] = `
<section
  className="player-finder-ctn"
  role="section"
>
  <header>
    <h1
      className="player-finder-ctn__title"
    >
      Football Player Finder
    </h1>
  </header>
  <form
    className="form-player-ctn"
    method="GET"
    onChange={[Function]}
    onSubmit={[Function]}
    role="search"
  >
    <div
      className="form-player-ctn__form-player-group"
    >
      <input
        className="form-control form-player-group__ctrl"
        name="name"
        placeholder="Player Name"
        required={true}
        type="text"
      />
    </div>
    <div
      className="form-player-ctn__form-player-group"
    >
      <select
        className="form-control form-player-group__ctrl"
        name="position"
      >
        <option
          value="Attacking Midfield"
        >
          Attacking Midfield
        </option>
        <option
          value="Central Midfield"
        >
          Central Midfield
        </option>
        <option
          value="Centre-Back"
        >
          Centre-Back
        </option>
        <option
          value="Centre-Forward"
        >
          Centre-Forward
        </option>
        <option
          value="Defensive Midfield"
        >
          Defensive Midfield
        </option>
        <option
          value="Keeper"
        >
          Keeper
        </option>
        <option
          value="Left Midfield"
        >
          Left Midfield
        </option>
        <option
          value="Left Wing"
        >
          Left Wing
        </option>
        <option
          value="Left-Back"
        >
          Left-Back
        </option>
        <option
          value="Right-Back"
        >
          Right-Back
        </option>
      </select>
    </div>
    <div
      className="form-player-ctn__form-player-group"
    >
      <input
        className="form-control form-player-group__ctrl"
        defaultValue="18"
        name="age"
        placeholder="Age"
        type="number"
      />
    </div>
    <div
      className="form-player-ctn__form-player-group"
    >
      <button
        className="btn btn-lg btn-block btn-primary form-player-group__btn"
        type="submit"
      >
        Search
      </button>
    </div>
  </form>
</section>
`;
